packages/rocketchat-ui-login/client/reset-password/resetPassword.js

```diff
@@ -44,10 +44,16 @@ Template.resetPassword.events({
 				RocketChat.Button.reset(button);
 				if (error) {
 					console.log(error);
-					swal({
-						title: t('Error_changing_password'),
-						type: 'error'
-					});
+					if (error.error === 'totp-required') {
```

Comment:
Does it throw this error even if successfully changing password?

Reply:
Yes, cuz after the change the process proceed to login, then we receive this error, if password was not changed the error will be different